### PR TITLE
feat: use isPossible instead of isValid to relax phone number validation

### LINKED ISSUE #1257: Loosen home number and international number validation

### PR DESCRIPTION
**Pull request by tshuli**
## Problem

Closes #1257

## Solution
Relaxes phone number validation by using isPossible() method instead of isValid() method.

From the [docs](https://www.npmjs.com/package/libphonenumber-js):
```
isPossible(): boolean
Checks if the phone number is "possible". 
Only checks the phone number length, doesn't check the actual phone number digits against any regular expressions.

isValid(): boolean
Checks if the phone number is "valid". 
First checks the phone number length and then checks the phone number digits against all available regular expressions.
``` 

Note that for international numbers, I've left the checks using the getType() method as a base level of validation to filter out numbers which should be clearly wrong, but we can go one step further and remove this check as well if we continue to get false negatives.

## Tests
- [ ] Create verified SMS field. Check that SMS can still be sent and received for valid Singapore number.

### CODE REVIEW

File: src/app/models/user.server.model.ts [hunk 1]
```diff
@@ -57,7 +57,7 @@ const compileUserModel = (db: Mongoose) => {
           validator: function (value: string) {
             const phoneNumber = parsePhoneNumberFromString(value)
             if (!phoneNumber) return false
-            return phoneNumber.isValid()
+            return phoneNumber.isPossible()
```
Comment (liangyuanruo) [inline src/app/models/user.server.model.ts:60]: does the issue to address include the form admin?

**Review by liangyuanruo — APPROVED** [review]
please confirm with @syan-syan on whether the issue is scoped to form submissions only, or is the admin's contact included. otherwise lgtm!

**Review by liangyuanruo — CHANGES_REQUESTED** [review]
Clarified with @syan-syan that there should be no change to the form admin's contact details.

**Comment by tshuli** [comment]
@liangyuanruo restored isValid check for admin numbers, for approval

**Review by liangyuanruo — APPROVED** [review]

**Pull request merged**